src/Network/Receive/ServerType0.pm

```diff
@@ -3014,73 +2286,6 @@ sub top10 {
 	}
 }
 
-sub top10_alchemist_rank {
```

Comment:
top10_alchemist_rank was removed from ServerType0 and added to Receive, but it's still in Sakexe_0